packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -508,6 +509,10 @@ export class AudiusAPIClient {
     this.env = env
   }
 
+  setIsReachable(isReachable: boolean) {
+    this.isReachable = isReachable
```

Comment:
Makes me slightly nervous to keep another separate state. We're risking bugs where it gets out of sync, but I think it's worth it.

Reply:
Yeah I wish I could reach into the redux state here but there isn't a clean way to do it with sagas that I could think of. I could export `store` from our top level `store.ts` file like we are doing with `dispatch` but then it creates this giant circular dependency that I would like to avoid